app/services/chat_notifications/providers/slack.rb

```diff
@@ -0,0 +1,19 @@
+class ChatNotifications::Providers::Slack
+  WEBHOOK_URL = ::BillingConfig['notifications']['webhook_url']
+  CHANNEL = ::BillingConfig['notifications']['room']
```

Comment:
room is specific for hipchat. Slack Webhooks are already associated to a channel so we should remove it from this provider